src/redux/utils.js

```diff
@@ -171,3 +173,20 @@ export const mapTransaction = (icpPrice, state) => trx => {
   };
   return transaction;
 };
+
+export const formatContact = contact => ({
+  image: contact?.emoji[0],
+  name: contact?.name,
+  id: contact?.value?.PrincipalId,
```

Comment:
why all the `?` checks? same in `formatContactForController`. we should think a way that if contact is null, the function wont be called. What u think

Reply:
TBH it always should exist, i've putted the ? when i was debugging, i can remove those if you want to